human-dynamics-estimator/src/HumanDynamicsEstimator.cpp

```diff
@@ -23,35 +34,132 @@ double HumanDynamicsEstimator::getPeriod()
 //---------------------------------------------------------------------
 bool HumanDynamicsEstimator::configure(yarp::os::ResourceFinder &rf)
 {
-    count = 0;
-    /*
-     * ------Human model loading and initialization
+    
+    /* ------Open a port:i for the human joint configuration
      */
-    modelFilename = rf.findFile("humanModel");
+    if (!m_humanJointConfiguration_port.open("/humanJointConfiguration:i"))
```

Comment:
You should first read the module name from the configuration file. 
That name should also act as prefix for all the ports opened by your module